javascript-closures/my-app/src/Counter.tsx

```diff
@@ -5,7 +5,7 @@ export function Counter() {
   useEffect(() => {
     getData();
     setCounter((prev) => prev + 1);
-  }, [getData, setCounter]);
+  }, [setCounter]);
```

Comment:
No, you still need getData in the array since it's a dependency. You should change the function getData to a useCallback.